server/routerlicious/packages/lambdas/src/nexus/index.ts

```diff
@@ -1047,15 +1047,9 @@ export function configureWebSocketServices(
 
 		/**
 		 * Message sent when a new signal is submitted to the router.
-		 *
-		 * Elements of 'contentBatches':
-		 * - may be individual or nested arrays.
-		 * - are `unknown` since the correct typing (1) is not validated and (2) doesn't need to be known to operate on it here.
-		 *
-		 * From debugging client end-to-end tests, we expect elements of `contentBatches` to be JSON-encoded strings.
-		 * These strings represent objects that conform to the ISignalEnvelope interface.
+		 * @see {@link DocumentDeltaConnection.emitMessages} for client-side emission of signals.
 		 */
-		socket.on("submitSignal", (clientId: string, contentBatches: unknown[]) => {
+		socket.on("submitSignal", (clientId: string, contentBatches: string[][]) => {
```

Comment:
`unknown` from before was better... for most of the reasons mentioned that are now deleted.

```suggestion
		socket.on(
			"submitSignal",
			/**
			 * @param contentBatches - typed as `unknown` array as it comes from wire and has not been validated.
			 * It is expected to be an array of strings (Json.stringified `ISignalEnvelope`s from
			 * [Container.submitSignal](https://github.com/microsoft/FluidFramework/blob/ccb26baf65be1cbe3f708ec0fe6887759c25be6d/packages/loader/container-loader/src/container.ts#L2292-L2294)
			 * and sent via
			 * [DocumentDeltaConnection.emitMessages](https://github.com/microsoft/FluidFramework/blob/ccb26baf65be1cbe3f708ec0fe6887759c25be6d/packages/drivers/driver-base/src/documentDeltaConnection.ts#L313C1-L321C4)),
			 * but actual content is passed-thru and not decoded.
			 */
			(clientId: string, contentBatches: unknown[]) => {
```